bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoHandler.java

```diff
@@ -195,19 +227,20 @@ public void onValueReceived(@Nullable String variable, @Nullable String value, @
 
         updateStatus(ThingStatus.ONLINE);
 
+        String oldValue = this.stateMap.get(variable);
         if (variable != null && value != null) {
             this.stateMap.put(variable, value);
```

Comment:
Any reason to add variables not supported to `stateMap`? They will not be used, just take up some memory. So the filtering in line 235 could be moved here.

Reply:
Might even be simpler if we can filter this in getWemoState… Will check later..

Reply:
Solved it with early return….